tests/functional/partial_parsing/test_partial_parsing.py

```diff
@@ -804,3 +805,31 @@ def test_pp_groups(self, project):
         )
         with pytest.raises(ParsingError):
             results = run_dbt(["--partial-parse", "run"])
+
+
+class TestPublicationArtifactAvailable:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "orders.sql": orders_sql,
+            "schema.yml": public_models_schema_yml,
+        }
+
+    def test_pp_publication_artifact_available(self, project):
+        # initial run with public model logs PublicationArtifactAvailable
+        results, log_output = run_dbt_and_capture(["--debug", "--log-format", "json", "run"])
```

Comment:
Does `parse` work here? Or it has to be `run`(which will be slightly slower).

Reply:
good call! updated